src/machine/machinectl.c

```diff
@@ -2636,6 +2708,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "format",          required_argument, NULL, ARG_FORMAT          },
                 { "uid",             required_argument, NULL, ARG_UID             },
                 { "setenv",          required_argument, NULL, 'E'                 },
+                { "number-ips",      required_argument, NULL, ARG_NUMBER_IPS      },
```

Comment:
I am not sure I like the name of this switch, as "ips" is really a sloppy term when referring to IP addresses. I'd suggest using: `--max-addresses` or at least `--max-addrs` or so...

Also, this new switch needs a man page update too!